src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -480,30 +467,27 @@ def utask_main(uworker_input):
     # We didn't crash at all. This might be a legitimately unreproducible
     # test case, so it will get marked as such after being retried on other
     # bots.
-    testcase = data_handler.get_testcase_by_id(uworker_input.testcase_id)
     data_handler.update_testcase_comment(testcase, data_types.TaskState.ERROR,
                                          'Unable to reproduce crash')
     task_creation.mark_unreproducible_if_flaky(testcase, True)
```

Comment:
Here for example, you will need to return something so postprocess marks it unreproducible.

Reply:
HMM ok im confused - the current minimize_task.py will return right after calling mark_unreproducible_if_flaky, so the steps that have been moved into postprocess wouldnt have been called anyway, no?
What did I change that makes us need to have the postprocess steps be called now?
Or is that something that we've been meaning to change that would be good to change now?

Reply:
>Or is that something that we've been meaning to change that would be good to change now?

This is an eventual change I'm talking about not something we need to do now.
But basically for this rearch, we need to move all database saving to postprocess.

Reply:
ahh ok yep! noted - ill get to dissecting these in a following change!